code/botlib/l_precomp.c

```diff
@@ -1323,7 +1323,7 @@ define_t *PC_DefineFromString(char *string)
 	script = LoadScriptMemory(string, strlen(string), "*extern");
 	//create a new source
 	Com_Memset(&src, 0, sizeof(source_t));
-	strncpy(src.filename, "*extern", MAX_PATH);
+	strncpy(src.filename, "*extern", sizeof(src.filename) - 1);
 	src.scriptstack = script;
```

Comment:
Shouldn't this ensure null terminator is also here?  strncpy by itself doesn't do this,